src/api/playground.js

```diff
@@ -67,7 +67,12 @@ const prove = async code => {
 };
 
 const compile = async code => {
-  const data = {code, action: ActionTypes.COMPILE_CONTRACT};
+  const data = {
+    code,
+    action: ActionTypes.COMPILE_CONTRACT,
+    compressed: false,
+    as_string: true
```

Comment:
What is this param?

Reply:
flag for returning the compiled contract as a string and not as an object (wrote it in the description)